src/WebJobs.Script/Binding/BlobBinding.cs

```diff
@@ -38,17 +38,25 @@ public override bool HasBindingParameters
 
             boundBlobPath = Resolve(boundBlobPath);
 
-            // TODO: Need to handle Stream conversions properly
-            Stream valueStream = context.Value as Stream;
-
-            Stream blobStream = context.Binder.Bind<Stream>(new BlobAttribute(boundBlobPath, FileAccess));
-            if (FileAccess == FileAccess.Write)
+            // TEMP: If the target type is set, use the binder logic to get a value of that type.
+            if (context.TargetType != null)
```

Comment:
Can we get rid of this check and always set TargetType? For non-C# languages, it's Stream (based on the 'else' block).